docs/contributing.md

```diff
@@ -42,7 +42,7 @@ message.
 
 When you address comments on a PR, don't make the changes in a commit on top (as
 is typical on GitHub). Instead, please make the changes in the appropriate
-commit. You can do that by checking out the commit (`jj checkout/new <commit>`)
+commit. You can do that by checking out the commit (`jj edit/new <commit>`)
 and then squash in the changes when you're done (`jj squash`). `jj git push`
```

Comment:
While you can use `jj edit` here, the next half of the sentence doesn't apply then. I think we can simply drop `checkout/` here. `jj checkout` is basically an alias for `jj new`

Reply:
is it better this way ?